### ISSUE TITLE
Unconstant behavior when outputting sixel data multiple times

### ISSUE THREAD

**Issue by yoichi** [arakiken/mlterm#4]
# Problem

When I output the sample sixel image described in https://en.wikipedia.org/wiki/Sixel multiple times,
the number of displayed images is unconstant.

[hi.txt](https://github.com/arakiken/mlterm/files/5433850/hi.txt)

![Screenshot 2020-10-25 at 07 04 25](https://user-images.githubusercontent.com/107537/97094557-aa1c1180-1690-11eb-9242-340de5b6e0bc.png)

# Expected behavior

I don't know the correct behavior, but I think it is correct to behave constantly.
I guess the correct behavior is to display it just once, or as many times as it is output.

# Environment

mlterm 3.8.6
Chromebook C101PA-OP1

```
yoichinakayama@penguin:~$ mlterm --version
mlterm version 3.8.6 
Features: otl ssh implugin imagelib(builtin) utmp
yoichinakayama@penguin:~$ uname -a
Linux penguin 5.4.67-09356-gf3ed4c0c4a31 #1 SMP PREEMPT Sun Sep 27 20:30:52 PDT 2020 aarch64 GNU/Linux
```

**Comment by arakiken** [comment]
Thanks for your detailed report, I fixed it.
https://github.com/arakiken/mlterm/commit/ac266c6e78837a4f84656266e991e1820a31bdb6

Original sixel graphics on VT340 etc supports drawing a single image in multiple sequences.
(See biplane.six in http://mlterm.sf.net/biplane.tar.gz)
Modern terminals including mlterm don't support it, but I added a dirty hack in mlterm to show biplane.six correctly.
This hack is the cause of the problem you reported.
